api/services/cardService.js

```diff
@@ -16,9 +16,8 @@ module.exports = {
             }
             return resolve(card);
           });
-      } else {
-        return Promise.reject({ message: 'Invalid arguments for findCard' });
       }
+      return Promise.reject({ message: 'Invalid arguments for findCard' });
```

Comment:
I don't think this is a blocker by any means, but we should file a follow up to refactor this function to use async await and the modern commands for game data.